utils/prometheus-node-exporter-lua/Makefile

```diff
@@ -166,6 +177,7 @@ endef
 
 $(eval $(call BuildPackage,prometheus-node-exporter-lua))
 $(eval $(call BuildPackage,prometheus-node-exporter-lua-nat_traffic))
+$(eval $(call BuildPackage,prometheus-node-exporter-lua-uci_dhcp_host))
```

Comment:
Please apply alphabetical ordering.

Reply:
Done, plus did a separate commit to order everything 
(on top of my previous commit) https://github.com/openwrt/packages/pull/12712/commits/c91554935f5adfc46fb047aba2401231ae8a077a ; fine with this ?